resources/js/stores/useAttributes.js

```diff
@@ -37,4 +37,8 @@ export const attributes = computedAsync(
     { lazy: true, shallow: false },
 )
 
+window.attributeLabel = (attributeCode) => {
+    return Object.values(attributes.value)?.find((attribute) => attribute.code === attributeCode)?.name
```

Comment:
I think we might want to return a computed value of this expression.
attributes is a lazy computed async. It will only be fetched when it's used and returns a default value in the meantime.

calling `attributeLabel` might give us undefined back while the request is still going and it will not be able to update.
if we wrap this in a computed function it has the chance to rerender with the updated attributes